templates/docs/base/code.md

```diff
@@ -60,6 +60,18 @@ View example of the code snippet
 View example of the code snippet
 </a></div>
 
+### Syntax highlighting
+
+We recommend using Prism for syntax highlighting in code snippet. Vanilla framework provides a built-in Prism theme, so only JavaScript of Prism library needs to be added to the page, and `language-*` class name to the respective code blocks.
+
+Make sure to include language support for any required languages or the `autoloader` plug-in (that automatically loads needed languages based on used class names).
```

Comment:
Should we mention _how_ to include plugins? Or link to https://prismjs.com/plugins/? It's not too hard to find how they've been included in the example, but it would be nice to point people in the right direction.

Reply:
Yes, good call. I forgot to link to the Prism site and docs.

Also, I figured worth mentioning you can manually add spans as well and not use Prism JS.